src/content/en/2022/jamstack.md

```diff
@@ -4,10 +4,12 @@ title: Jamstack
 #TODO - Review and update chapter description
 description: Jamstack chapter of the 2022 Web Almanac covers adoption of technology, performance of websites built with Jamstack and weights for various resources.
 authors: [seldo, whitep4nth3r]
-reviewers: [GregBrimble, Nutlope]
-analysts: [seldo, GregBrimble]
-editors: []
+reviewers: [GregBrimble, Nutlope, tunetheweb]
+analysts: [seldo, tunetheweb]
+editors: [DesignrKnight]
```

Comment:
@seldo @whitep4nth3r could you confirm these are the contributors, listed in descending order of contribution? and that we haven't missed anyone?

Reply:
No one is missing, to my knowledge!

Reply:
Did @GregBrimble and @Nutlope contribute? Can't remember now but, if not, then we should remove.

Reply:
@tunetheweb I did not contribute actually, feel free to remove me.

Reply:
Done. Thanks for letting us know. Maybe next year!

Reply:
Likewise, I couldn't this year either unfortunately.